pulp.spec

```diff
@@ -389,6 +388,12 @@ Requires: Django14
 %else
 Requires: python-django >= 1.4.0
 %endif
+# RHEL6 or RHEL7
+%if 0%{?rhel} == 6 || 0%{?rhel} == 7
+Requires: python-celery == 3.1.17
+%else
+Requires: python-celery >= 3.1.12
```

Comment:
That's a good idea! It does not allow someone to use celery 3.1.12 - 3.1.16 but the probability that it is needed is very low I guess.